tests/unity/wrap_test/src/fns/fns.h

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/*
+	DO NOT FORMAT THIS FILE!
+
+	It contains function declarations that is used to validate the test wrapper
+	scripts (func_name_list.py and header_parse.py) which renames the functions.
+	The regular expression has had several bugs in the past, and the formating
+	of the functions here are used to validate that	the fixes still work when
+	modifying the regex.
+
+	List of fixes this file tries to reproduce:
+	https://github.com/nrfconnect/sdk-nrf/pull/1413
+	https://github.com/nrfconnect/sdk-nrf/pull/2182
+	https://github.com/nrfconnect/sdk-nrf/pull/3432
+	https://github.com/nrfconnect/sdk-nrf/pull/5069
+	https://github.com/nrfconnect/sdk-nrf/pull/5814
+	https://github.com/nrfconnect/sdk-nrf/pull/6478
+	https://github.com/nrfconnect/sdk-nrf/pull/6547
+*/
+
+#ifndef __FNS_H
+#define __FNS_H
+
+/* #include <toolchain/common.h> */
```

Comment:
Remove?

Reply:
I think it's verifying that the script can indeed test a commented out `include` line 😉

Reply:
Just got that :-D